scripts/testpop/statter.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/cactus/go-statsd-client/statsd"
+)
+
+// WriteCloserToSender is an adapter from io.WriteCloser to statsd.Sender
+type WriteCloserToSender struct {
```

Comment:
I suggest we change the name to smth like "FileSender", because I read current version as "write closer to (= not far from) sender".